governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+# Maker Executive Aug 2 2023
+
+# [Executive Proposal] DSR Increase, Spark D3M Changes, Harbor Trade Credit (RWA004-A) Changes, New Silver (RWA002-A) Changes, Launch Project Funding, AVC Member Participation Rewards, Clydesdale Quarterly Surplus Return - August 2, 2023
+
+The Governance Facilitator(s), AmusingAxl, Sidestream and Arby have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- Increase the Dai Savings Rate (DSR) by 4.81% from 3.19% to 8%.
+- Increase the DIRECT-SPARK-DAI Maximum Debt Ceiling by 180 million DAI from 20 million DAI to 200 million DAI, as described below.
+- Set Spark Protocol DAI market Interest Strategy to track the ETH-C stability fee instead of the DSR rate as described below.
+- Set Spark Protocol DAI market Loan-To-Value (LTV) and Liquidation Threshold to 0.01% as described below.
+- Set Spark Protcol WETH market Reserve Factor to 5% as described below.
+- Set Spark Protocol WETH market Variable Rate Slope 1 to 3% as described below.
+- Decrease Harbor Trade Credit Vault (RWA004-A) Debt Ceiling as detailed below.
+- Increase New Silver (RWA002-A) Debt Ceiling and Stability Fee as described below.
+- A MakerDAO Resolution will be approved to initiate a Clydesdale quarterly surplus return as described below.
+- Distribute **2,000,000 DAI** for Launch Project funding as described below.
+- Distribute **134.1 MKR** for Q2 AVC Member Participation Rewards as described below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
```

Comment:
Should match terminology with executive summary. ie use 'changes' there then should use 'changes' here, rather than 'changes and additions'